randovania/games/super_metroid/json_data/Maridia.txt

```diff
@@ -936,49 +936,49 @@ Extra - asset_id: 158
 > Door to Halfie Climb Room; Heals? False; Default Node
   * Layers: default
   * Normal Door to Halfie Climb Room/Door to Colosseum
-  > Door to Colosseum Save Station
+  > Door to Colosseum Save Room
       All of the following:
           Gravity Suit
           Any of the following:
               Space Jump
               Grapple Beam and Hi-Jump Boots
 
-> Door to Colosseum Save Station; Heals? False
+> Door to Colosseum Save Room; Heals? False
   * Layers: default
-  * Missile Door to Colosseum Save Station/Door to Colosseum
+  * Missile Door to Colosseum Save Room/Door to Colosseum
   > Door to Halfie Climb Room
       All of the following:
           Gravity Suit
           Any of the following:
               Space Jump
               Gravity Suit and Hi-Jump Boots
-  > Door to Precious Room
+  > Door to The Precious Room
       Trivial
 
```

Comment:
TIL this one was named because of a typo according to https://wiki.supermetroid.run/The_Precious_Room

Reply:
yeah its pretty funny